addons/l10n_ar/report/invoice_report_view.xml

```diff
@@ -10,7 +10,7 @@
                 <field name="l10n_ar_state_id"/>
                 <filter name="with_document" string="With Document" domain="[('l10n_latam_document_type_id', '!=', False)]"/>
             </search>
-            <filter name="user" position="after">
+            <filter name="company" position="after">
```

Comment:
Why this change?

Reply:
That was to put the group by company before the group by state and account, but actually it wasn't necessary. We achieve this with the context priorities.